src/GLPIMailer.php

```diff
@@ -208,30 +214,22 @@ public function send()
             $this->email->html($this->normalizeLineBreaks($html_body));
         }
 
-        $debug = null;
         try {
             $this->error = null;
             $this->email->ensureValidity();
             $sent_message = $this->transport->send($this->email);
-            $debug = $sent_message->getDebug();
+            $this->debug = $sent_message->getDebug();
             return true;
         } catch (\Symfony\Component\Mailer\Exception\TransportExceptionInterface $e) {
             $this->error = $e->getMessage();
-            $debug = $e->getDebug();
+            $this->debug = $e->getDebug();
         } catch (\LogicException $e) {
             $this->error = $e->getMessage();
         } catch (\Throwable $e) {
             $this->error = $e->getMessage();
             ErrorHandler::getInstance()->handleException($e, true);
         }
 
-        if ($debug !== null && $_SESSION['glpi_use_mode'] == Session::DEBUG_MODE) {
-            Toolbox::logInFile(
-                'mail-debug',
-                ('# ' . ($this->debug_header_line ?? __('Sending email...')) . "\n") . $debug
-            );
-        }
```

Comment:
While working on this feature, I considered that as soon as the debug log would be displayed in the UI during the sending test, the use of the `mail-debug` log file would be very rare, and that we could delete it.

@orthagh Are you OK with this ?

Reply:
Yes